docs/src/getting-started/installation.md

```diff
@@ -46,15 +46,37 @@ First, add the Starknet Foundry plugin to asdf:
 asdf plugin add starknet-foundry
 ```
 
-Install the latest version:
+#### Version Management
+
+- Common Error
+
+Users may encounter this error when trying to use `snforge` or `sncast` without setting a version:
+
+```shell
+No version is set for command snforge
+Consider adding one of the following versions in your config file at starknet-foundry 0.32.0
+```
+
+The error indicates that `Starknet Foundry` version is unset. Set the version as outlined to fix this
+
+#### Switching Versions
+
+To switch between different versions:
 
 ```shell
-asdf install starknet-foundry latest
+# Install a different version
+asdf install starknet-foundry 0.32.0
+
+# Switch global version
+asdf global starknet-foundry 0.32.0
+
+# Or switch local version
+asdf local starknet-foundry 0.32.0
 ```
 
-See [asdf guide](https://asdf-vm.com/guide/getting-started.html) for more details.
+"For more details on using `asdf` see this [asdf](https://asdf-vm.com/guide/getting-started.html#_6-set-a-version)"
```

Comment:
I think we can still simplify this. 

1. Remove `Version Management` heading
2. Only write that if the `No version is set for command snforge...` error is set, user needs to set it through asdf
3. Only show usage of `asdf global` and link to asdf docs after this